src/extension/background-script/index.ts

```diff
@@ -158,7 +158,7 @@ browser.runtime.onInstalled.addListener(handleInstalled);
 
 console.info("Welcome to Alby");
 init().then(() => {
-  if (isFirstInstalled) {
+  if (isFirstInstalled && !state.getState().accounts) {
```

Comment:
accounts is `{}` which is truthy, isn't it? so I'd do the same check as here: 

https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/setup/status.js#L6-L7

Reply:
Done!